### ISSUE TITLE
Incorrect path passed to getsize and abspath

### ISSUE THREAD

**Issue by pisarik** [pycabook/fileinfo#2]
### Description

Shouldn't `get_info` method pass `self.original_path` instead of `self.filename` to `abspath` and `getsize`? Otherwise If the file not in the current directory, then `get_info` will fail.

But I am glad that according to principles you describe in your book, `abspath` and `getsize` are external actors, we are querying them and therefore should check parameters, which we pass to them. And it would catch this bug.

We need to add to `test_get_info` following asserts:

```
abspath_mock.assert_called_with(original_path)
getsize_mock.assert_called_with(original_path)

```

**Comment by lgiordani** [comment]
@pisarik apologies for leaving this unanswered for a long time. Thanks for the issue and for the PR, I didn't ignore you. I will merge it shortly. Thanks!

**Comment by lgiordani** [comment]
Hi! I just published version 1.0.12 of the book with your contribution. Thanks a lot! I have to close the PR without merging it because I had to reconcile the tags and the history of the repository, so that was merged piece by piece. Thanks again!

**Comment by pisarik** [comment]
No problem, you are very courteous. Thank you for your work!:)
